focuspoints.lrdevplugin/CanonDelegates.lua

```diff
@@ -0,0 +1,129 @@
+--[[
+  Copyright 2016 Joshua Musselwhite, Whizzbang Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--]]
+
+--[[
+  A collection of delegate functions to be passed into the DefaultPointRenderer when
+  the camera is Canon
+--]]
+
+local LrStringUtils = import "LrStringUtils"
+require "Utils"
+
+CanonDelegates = {}
+
+--[[
+-- metaData - the metadata as read by exiftool
+--]]
+function CanonDelegates.getAfPoints(photo, metaData)
+  local imageWidth = ExifUtils.findFirstMatchingValue(metaData, { "AF Image Width", "Exif Image Width" })
+  local imageHeight = ExifUtils.findFirstMatchingValue(metaData, { "AF Image Height", "Exif Image Height" })
+  if imageWidth == nil or imageHeight == nil then
+    return nil
+  end
+
+  local orgPhotoWidth, orgPhotoHeight = parseDimens(photo:getFormattedMetadata("dimensions"))
+  local xScale = orgPhotoWidth / imageWidth
```

Comment:
can you describe what is going on here? I would expect imageWidth and orgPhotoWidth to be the same values.